etc/profile-m-z/zoom.profile

```diff
@@ -20,14 +22,20 @@ whitelist ${HOME}/.cache/zoom
 whitelist ${HOME}/.config/zoomus.conf
 whitelist ${HOME}/.zoom
 include whitelist-common.inc
+include whitelist-var-common.inc
```

Comment:
Can we add `whitelist-runuser-common` and `whitelist-usr-share-common`?

Reply:
we can, but why? It works fine without that :)

Reply:
Why using firejail? It works without it ;)

FYI: `include whitelist-runuser-common.inc` means only a few listed sockets in /run/user/UID are allowed. W/O it everything is allowed. Same for wusc.

Reply:
You can copy it, in 0.9.62 it may break XWayland on GNOME. But if you are still on X11 or KDE you can test it.